styles.css

```diff
@@ -0,0 +1,126 @@
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+  
+}
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+.projects-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  grid-gap: 4rem;
+  width: 100%;
+  max-width: 1280px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+}
+@media(max-width: 480px) 
+{
+  #h1
+  {
+      font-size: 1.8em;
+  }
+}
+
+#projects
+{
```

Comment:
Awesome job! Very detailed stylesheet, however, the opening bracket should always be on the same line as the opening of the rule